src/components/AccountContent/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import ProductAvailable from './ProductAvailable'
+import Review from './Review'
+
+const AccountContent = () => {
```

Comment:
Regarding the design, this section carries about products and reviews without an account or user information. So, would you mind considering another class name like AccountRelatedProduct or do you have ideas in mind?

Reply:
Since this is the container class for products and reviews, I personally think that reviews are not part of `AccountRelatedProduct`. What if we use `AccountRelatedProduct` as a class containing `Available products` and `Out of stock products` instead?